aspnet/fundamentals/distributed-cache.rst

```diff
@@ -1,10 +1,106 @@
+
+.. literalinclude:: dist-cache/sample/src/DistCacheSample/Startup.cs
+  :language: c#
+  :linenos:
+  :lines: 27-40
+  :emphasize-lines: 8-13
+
+.. note:: To install Redis on your local machine, install the chocolatey package http://chocolatey.org/packages/redis-64/ and run ``redis-server`` from a command prompt.
+
+Using a SQL Server Distributed Cache
+------------------------------------
+The `SqlServerCache <https://github.com/aspnet/Caching/tree/1.0.0-rc1/src/Microsoft.Extensions.Caching.SqlServer>`_ implementation allows the distributed cache to use a SQL Server database as its backing store. The installation script installs a table with whatever name you configure and the following schema:
+
+.. image:: dist-cache/_static/SqlServerCacheTable.png
+
+.. note:: If you're working with the RC1 version of ``SqlServerCache``, there isn't a working installer. You can install the required table using the scripts found in the `SqlQueries.cs file <https://github.com/aspnet/Caching/blob/1.0.0-rc1/src/Microsoft.Extensions.Caching.SqlConfig/SqlQueries.cs>`_. This will be addressed in RC2.
+
+Just as with a Redis implementation, working with a ``SqlServerCache`` implementation should be transparent to your application code. Your application code should simply request an instance of type ``IDistributedCache`` and get and set cached values through this interface. You should only configure ``SqlServerCache`` in your application's ``ConfigureServices`` method.
+
+In the sample code associated with this article, a ``SqlServerCache`` implementation is used when the server is configured for a ``Production`` environment. Thus, ``ConfigureProductionServices`` has the required code to configure the ``SqlServerCache``:
+
+.. literalinclude:: dist-cache/sample/src/DistCacheSample/Startup.cs
+  :language: c#
+  :linenos:
+  :lines: 42-56
+  :emphasize-lines: 9-14
+
+.. note:: The ``ConnectionString`` (and optionally, ``SchemaName`` and ``TableName``) should typically be stored in :doc:`configuration` in real-world applications, rather than hard-coded here where a code change is required to update the values.
+
+Recommendations
+---------------
+When deciding which implementation of ``IDistributedCache`` is right for your application, choose between Redis and SQL Server based on your existing infrastructure and environment and your team's experience. If your team is more comfortable working with Redis, it's an excellent choice. If your team prefers SQL Server, you can be confident in that implementation as well. Avoid using the in-memory implementation (``MemoryCache``) in production environements.
+
+.. warning:: The in-memory implementation of ``IDistributedCache`` is only provided for testing purposes and should not be used in production.
```

Comment:
Not quite. Its ok to use for things like session if you only have one server instance, which will be common for small sites.